internal/pkg/manifest/workload.go

```diff
@@ -64,7 +64,7 @@ type WorkloadManifest interface {
 	Validate() error
 	RequiredEnvironmentFeatures() []string
 	Load(sess *session.Session) error
-	Manifest() interface{}
+	Manifest() any
```

Comment:
what do you think of ⬇️ 
```suggestion
	Manifest() workloadManifest
```

Reply:
I think it is probably fine to leave as `any` since it really needs the manifest itself rather than the interface. It doesn't make sense for a public method to return a private interface (because the client in the CLI package can't call any of the private methods there). The reason why we need `workloadManifest` in the dynamic struct is because we need to call the method under the hood.